### PR TITLE
Add rfc to remove an TODO: that seems to be referencing the wrong struct field commit.c:360

### PR DESCRIPTION
**Pull request by skinnyal**
Thanks for taking the time to contribute to Git!

Those seeking to contribute to the Git for Windows fork should see
http://gitforwindows.org/#contribute on how to contribute Windows specific enhancements.

If your contribution is for the core Git functions and documentation
please be aware that the Git community does not use the github.com issues
or pull request mechanism for their contributions. 

Instead, we use the Git mailing list (git@vger.kernel.org) for code and 
documenatation submissions, code reviews, and bug reports. The
mailing list is plain text only (anything with HTML is sent directly
to the spam folder). 

Nevertheless, you can use submitGit to conveniently send your Pull
Requests commits to our mailing list.

Please read the "guidelines for contributing" linked above!

### CODE REVIEW

**Comment by skinnyal** [comment]
Was doing some digging within the code, and came across this. @dscho

File: commit.c [hunk 1]
```diff
@@ -358,7 +358,12 @@ void release_commit_memory(struct commit *c)
 	free_commit_buffer(c);
 	free_commit_list(c->parents);
 	/* TODO: what about commit->util? */
+	/*
+	It seems as if we can remove this todo. util seems to not be a field of the struct commit,
+	definition of struct commit --> commit.c:27
+	The commit list also, does not contain a field for util.
 
+	*/
```
Comment (dscho) [inline commit.c]: Good point.

Let's just remove the line? Maybe you can identify the commit that forgot to remove it? (Hint: `git log -L <argument>` is your friend...)

Once you identify the commit, you could imitate the commit message of 60129c61e55fa5567d8b7ebe21b607e363330f07:

- the lines wrap at 76 columns/line
- the first line is a concise title
- the commit message gives a compelling reason for the change, answering the "why" (the patch already explains the "how")

Comment (skinnyal) [reply commit.c]: Sounds good. Will do asap.

**Comment by skinnyal** [comment]
hey @dscho went ahead, and made the small change. Have been behind on some project work, but wanted to make sure that I had this checked in. 

Would I just leave this here, or send this to the mailing list?

-Al

**Comment by skinnyal** [comment]
Hey @dscho I'll work on this right now. Sorry about the delay school, and things got hectic.

**Comment by dscho** [comment]
@skinnyal could you do a `git rebase -i` on top of upstream's `master`, squashing all the iterative changes into a single commit/patch?

**Comment by skinnyal** [comment]
@dscho I went ahead, and rebased. I think I did it in the correct order. I'm unsure if the merging to my local remote should be the latest entry in `git log` , or no.

**Comment by dscho** [comment]
>  I'm unsure if the merging to my local remote should be the latest entry in `git log` , or no.

Ideally, there should not even be a merge commit in your branch. There is no way to "export" a merge commit via `format-patch`. It cannot be represented as a patch.

**Comment by dscho** [comment]
> Ideally, there should not even be a merge commit in your branch.

To clarify what I mean: this is what I am seeing in the `Commits` tab of this PR:

* Add rfc to remove an TODO: that seems to be referencing the wrong str… 
* TODO doc: remove unneeded TODO for release_commit_memory 
* doc: remove unneeded TODO for release_commit_memory 
* Merge branch 'skinnyal/microprojects' of https://github.com/skinnyal/git
* doc: remove unneeded TODO for release_commit_memory
* Merge branch 'skinnyal/microprojects' of https://github.com/skinnyal/git

What I expected to see would look more like

* doc: remove unneeded TODO for release_commit_memory

This could be achieved by running these commands:

```sh
git fetch https://github.com/git/git master
git rebase -i FETCH_HEAD
```

and then changing each `pick` except the very first one to `squash`. Then save the file and close the editor (or editor tab), and when the editor comes back up, edit the commit message appropriately (most likely just keeping the last commit message).

**Comment by skinnyal** [comment]
Ah, Okay. I think it should be fixed now @dscho .

**Comment by skinnyal** [comment]
No problemo! Thanks for following up with this. Sent it off to the mailing list!

**Comment by dscho** [comment]
For lurkers: the mail is here: https://public-inbox.org/git/CACsJy8D_gTKWXogPDNW7NQk_a0ChBu28HfGu388hFn3-by_cRw@mail.gmail.com/T/#t

**Pull request closed without merging**